### ISSUE TITLE
Turn off dropout during prediction

### ISSUE THREAD

**Issue by zsong** [migueldeicaza/TensorFlowSharp#225]
Hi, I compared same model running on TensorFlowSharp and Keras in Python, but I got most results differs by less than 1%.

Keras should have dropout automatically turned off during the inference, but is there a way to turn off dropout in TensorFlowSharp?
